### ISSUE TITLE
Replace broken link

### ISSUE THREAD

**Issue by erictleung** [freeCodeCamp/how-to-contribute-to-open-source#301]
Broken link:
https://medium.com/@kentcdodds/first-timers-only-78281ea47455

Working link:
https://kentcdodds.com/blog/first-timers-only

```diff
  - [How to find your first Open Source bug to fix](https://medium.freecodecamp.org/finding-your-first-open-source-project-or-bug-to-work-on-1712f651e5ba#.slc8i2h1l) by [@Shubheksha](https://github.com/Shubheksha)
- - [First Timers Only](https://medium.com/@kentcdodds/first-timers-only-78281ea47455) by [@kentcdodds](https://github.com/kentcdodds)
+ - [First Timers Only](https://kentcdodds.com/blog/first-timers-only) by [@kentcdodds](https://github.com/kentcdodds)
  - [Bring Kindness Back to Open Source](http://www.hanselman.com/blog/BringKindnessBackToOpenSource.aspx) by [@shanselman](https://github.com/shanselman)
```

All `README` files need to be fixed (I believe all of them have this broken link).

**Comment by preludek** [comment]
I would like to take this issue.

**Comment by erictleung** [comment]
@preludek go for it! Be sure to add `Closes #301` in the pull request message body.
